test/unit/alignment/pairwise/global_affine_unbanded_collection_simd_test.cpp

```diff
@@ -17,24 +17,70 @@
 namespace seqan3::test::alignment::collection::simd::global::affine::unbanded
 {
 
-static auto dna4_01 = []()
+static auto dna4_all_same = []()
```

Comment:
```suggestion
static auto dna4_all_same_length = []()
```
do you mean that? or that the sequences have the same nucleotides?

Reply:
All sequences are the same